client/js/app/actions/ExplorerActions.js

```diff
@@ -3,10 +3,10 @@ var AppDispatcher = require('../dispatcher/AppDispatcher');
 var ExplorerConstants = require('../constants/ExplorerConstants');
 var ExplorerStore = require('../stores/ExplorerStore');
 var ExplorerValidations = require('../validations/ExplorerValidations');
-var ValidationUtils = require('../utils/ValidationUtils');
-var ExplorerUtils = require('../utils/ExplorerUtils');
 var NoticeActions = require('./NoticeActions');
 var AppStateActions = require('./AppStateActions');
+var RunValidations = require('../utils/RunValidations');
```

Comment:
:D I commented in the wrong file:

https://github.com/keen/explorer/pull/56/files#diff-dca69ab628f9de688bfff269868227e4R6